src/tools/clippy/tests/ui/checked_unwrap/simple_conditionals.rs

```diff
@@ -172,6 +172,7 @@ fn issue11371() {
     static mut X: Option<i32> = Some(123);
     unsafe {
         if X.is_some() {
+            //~^ ERROR: creating a shared reference to mutable static is discouraged
```

Comment:
Since this test is unrelated, we just want to fix this.  E.g., we could say `if let Some(_) = x`.